05 - coder-files/src/App.jsx

```diff
@@ -1,16 +1,13 @@
+const ANIMALS = ["dog", "cat", "chicken", "cow", "sheep", "horse"];
+
 export default function App() {
   return (
-    <div className="flex gap-10 justify-center flex-wrap px-10 text-blue-600 mt-10">
-      <a href="/">Home</a>
-      <a href="/" target="_blank">Exercise 1</a>
-      <a href="/" target="_blank">Exercise 2</a>
-      <a href="/" target="_blank">Exercise 3</a>
-      <a href="/" target="_blank">Exercise 4</a>
-      <a href="/" target="_blank">Exercise 5</a>
-      <a href="/" target="_blank">Exercise 6</a>
-      <a href="/" target="_blank">Exercise 7</a>
-      <a href="/" target="_blank">Exercise 8</a>
-      <a href="/" target="_blank">Exercise 9</a>
+    <div className="flex gap-10 justify-center flex-wrap px-10 mt-10">
+      <ul className="list-disc">
+        {ANIMALS.map((animal) => (
+          <li key={new Date().getTime() + animal}>{animal}</li>
```

Comment:
Hey @riyoneri, This is a Clever way to add a key not using an index but creating your identifier. Kudos